src/App.js

```diff
@@ -39,6 +62,7 @@ function App() {
         let copy = [...toDoList];
         copy = [...copy, { id: toDoList.length + 1, task: userInput, complete: false }];
```

Comment:
Can you make the id value randomized? Right now there is a probability to have two tasks with the same ID since IDs are just numbers 1,2,3...
When adding a new task, instead create an ID that is a string and that contains a random value.